hazenlib/utils.py

```diff
@@ -222,17 +252,16 @@ def get_rows(dcm: pydicom.Dataset) -> float:
 
 
 def get_columns(dcm: pydicom.Dataset) -> float:
-    """
-    Returns number of image columns (columns)
+    """Get the Columns field from the DICOM header
 
-    Parameters
-    ----------
-    dcm: pydicom.Dataset
+    Args:
+        dcm (pydicom.Dataset): DICOM image object
 
-    Returns
-    -------
-    columns: float
+    Returns:
+        float: value of the Columns field from the DICOM header, or defaults to 256
     """
+    # TODO: explore what type of DICOM files do not have Columns in DICOM header
+    # check with physicists whether 256 is an appropriate default value
```

Comment:
As per the NEMA DICOM standards, all MR DICOMs should have Rows and Columns entries. 256 is reasonable default for both.